python/pyarrow/_fs.pyx

```diff
@@ -782,6 +782,9 @@ cdef class LocalFileSystem(FileSystem):
     use_mmap : bool, default False
         Whether open_input_stream and open_input_file should return
         a mmap'ed file or a regular file.
+    reuse : bool, default True
+        If set to False, will use posix_fadvise to (try) not use the page cache.
+        This will only work on Linux!!
```

Comment:
```suggestion
        If set to False, will use posix_fadvise to (try) not use the page cache.
        This will only work on Linux.
```

Why `(try)`?  Also, we should make this comment consistent with the one in the C++.  The user should be advised this will have a negative impact on performance.  The user should be informed that they might want to do this to prevent swapping out important information.

Reply:
" The user should be informed that they might want to do this to prevent swapping out important information." I am not sure what this means

Reply:
Also I am not sure what you mean by make this comment consistent with the one in C++. Do you mean I should use the same words?

Reply:
By the one in C++ I meant whatever comment you added to respond to this suggestion: https://github.com/apache/arrow/pull/13662/files#r929417116